plugins/inputs/bigbluebutton/testdata/getMeetings.xml.empty_state

```diff
@@ -0,0 +1,5 @@
+<response>
+    <returncode>FAILED</returncode>
+    <messageKey>checksumError</messageKey>
+    <message>You did not pass the checksum security check</message>
+</response>
```

Comment:
A newline at the end of the file cannot hurt and eases the life on the console. :-)